GrandCentralBoard/Widgets/Harvest/HarvestSource.swift

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
`HarvestSource` uses proprietary product name... and we can change the service in the future... Maybe `TimeTrackingSource`?